go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/pabowers/statsd
```

Comment:
why are you changing the module path?

Reply:
that was for local testing, you are right that I shouldn't change here and I also need to remove the gomodule references

Reply:
I have corrected the module path to correspond with this module